apps/end2end/clock/end2end_stopwatch.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+var Clock = require('./regions/app_view');
+var expect = require('chai').expect;
+
+marionette('Clock Acceptance Suite', function(){
+    var client;
+    var clock;
+    client = marionette.client({
+        prefs: {
+            //Prefs go here
+        },
+        settings: {
+            'lockscreen.enabled': false
```

Comment:
In gaia_test.py, we have some DEFAULT_SETTINGS and DEFAULT_PREFS that we set for every test. Perhaps we need something similar for MarionetteJS UI testing?